### DISCUSSION TITLE
The list on my phone dont appear in the laptop

### DISCUSSION THREAD

**Discussion by Sulaiman-alkadi** [community/community#30253] [Mobile]
I dont no if its sync problems or github doesn't support such a thing .

**Comment by v1ctorio** [comment]
I don't really understand what you mean but GitHub mobile has far fewer features, if you want more use the web mobile version instead of the app

**Comment by NikhilMath** [comment]
Mobile is less effective than Desktop, 
Don't ask me why they did it like that, it is just less powerful sadly.

**Comment by alialhammad1992** [comment]
ios 

The mobile application should be GithHub better than the mobile browser, but unfortunately they are slow in developing an application. Please pay more attention to the application and make its advantages more than the mobile browser.

**Comment by jfullstackdev** [comment]
open Chrome (or other supported browsers) on your mobile and log in to GitHub, this is I think the workaround to see features like on the Desktop
